mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/tiltaksgjennomforinger/TiltaksgjennomforingValidator.kt

```diff
@@ -195,14 +194,48 @@ class TiltaksgjennomforingValidator(
         return errors.takeIf { it.isNotEmpty() }?.left() ?: dbo.right()
     }
 
-    private fun MutableList<ValidationError>.validateKursTiltak(dbo: TiltaksgjennomforingDbo) {
-        if (dbo.fremmoteTidspunkt != null && dbo.fremmoteTidspunkt.toLocalTime() == LocalTime.of(0, 0)) {
-            add(
-                ValidationError(
-                    name = "fremmoteTid",
-                    message = "Fremmøte klokkeslett kan ikke være kl 00:00",
-                ),
-            )
+    private fun MutableList<ValidationError>.validateKursTiltak(avtale: AvtaleAdminDto, dbo: TiltaksgjennomforingDbo) {
+        if (Tiltakskoder.isKursTiltak(avtale.tiltakstype.arenaKode)) {
+            if (dbo.fremmoteTidspunkt != null && dbo.fremmoteTidspunkt.toLocalTime() == LocalTime.of(0, 0)) {
+                add(
+                    ValidationError(
+                        name = "fremmoteTid",
+                        message = "Fremmøte klokkeslett kan ikke være kl 00:00",
+                    ),
+                )
+            }
+
+            if (dbo.deltidsprosent == null) {
+                add(
+                    ValidationError.of(
+                        TiltaksgjennomforingDbo::deltidsprosent,
+                        "Deltidsprosent er påkrevd",
+                    ),
+                )
+            } else if (dbo.deltidsprosent <= 0) {
+                add(
+                    ValidationError.of(
+                        TiltaksgjennomforingDbo::deltidsprosent,
+                        "Deltidsprosent må være større enn 0",
+                    ),
+                )
+            } else if (dbo.deltidsprosent > 100) {
+                add(
+                    ValidationError.of(
+                        TiltaksgjennomforingDbo::deltidsprosent,
+                        "Deltidsprosent kan ikke være større enn 100",
+                    ),
+                )
+            }
+        } else {
+            if (dbo.deltidsprosent != null) {
```

Comment:
Har antatt at denne verdien kun skal settes for kurstiltak, men stemmer nok ikke. Fra Arena har alle gjennomføringer en verdi. Denne sjekken burde nok fjernes.